mods/persistence/modules/projectiles/overrides.dm

```diff
@@ -0,0 +1,38 @@
+/obj/item/gun/launcher/syringe/rapid
+	origin_tech = "{'wormholes':1337}"
+
+/obj/item/gun/energy/temperature
+	origin_tech = "{'wormholes':1337}"
+
+/obj/item/grenade/anti_photon
```

Comment:
Ditto

Reply:
didnt realize this was a grenade lol 
removed from override